test/distributed/tensor/parallel/test_tp_style.py

```diff
@@ -148,7 +178,13 @@ def test_rowwise_parallel_embedding(self):
 
 
     @with_comms
-    def test_prepare_module_input(self):
+    @run_with_both_funcol_impls_with_arg
```

Comment:
instead of doing a custom wrapper for the purpose of switching c10d_functional, wondering if we should just use the parameterization in tests?

```
@parametrize(funcol_namespace, [torch.ops._c10d_functional, torch.ops.c10d_functional])
def test_xxx(funcol_namespace):
    ....
```